src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -17,7 +17,8 @@
     protected static final long TIMEOUT = 30;
     protected static final long LONG_TIMEOUT = 120;
     private static final int MAX_RETRYCOUNT = 8;
-    private static final int SLEEP_TIME = 5000;
+    private static final int COUNT = 3;
```

Comment:
What's this counting?

Reply:
Yes, I get what it's actually doing. What I'm trying to point out is that the variable name `COUNT` is not descriptive at all. It's not actually counting anything, and even if it were, it'd be much more useful to know what it was counting